docs/conf.py

```diff
@@ -12,18 +12,28 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 
+import importlib.util
+from pathlib import Path
+
 import sphinx_rtd_theme
 
-import deeppavlov
+# Import __version__ from deeppavlov may cause issues in the process of
+# development: __version__ is taken from _meta.py which is updated only after
+# running setup.py. A developer may be puzzled why the version change is not
+# reflected in the documentation (if he didn't run installation after that).
+# We can't use relative import from setup.py, therefore forced to use importlib.
+spec = importlib.util.spec_from_file_location('setup', Path(__file__).resolve().parents[1] / 'setup.py')
+setup = importlib.util.module_from_spec(spec)
+spec.loader.exec_module(setup)
 
 # -- Project information -----------------------------------------------------
 
 project = 'DeepPavlov'
-copyright = '2018, ' + deeppavlov.__author__
-author = deeppavlov.__author__
+copyright = f'2018, {setup.__author__}'
+author = setup.__author__
```

Comment:
This doesn't feel right...

Reply:
Only change 2018 -> 2020?

Reply:
Importing `setup` in docs doesn't feel right for some reason (=